### DISCUSSION TITLE
How do you find the exact x,y position?

### DISCUSSION THREAD

**Discussion by Logon27** [motion-canvas/motion-canvas#730] [Q&A]
Hello, I was wondering if there is any way in the preview window to be able to find the exact x and y coordinates of my mouse in the scene? I want to move an element to a specific position, but I really do not want to have to keep guessing and checking if I am moving to the proper location (x and y coordinates).

**Comment by aarthificial** [comment]
The ability to copy the current mouse position was added in #737
